metadata/src/main/resources/common/metadata/ZkMigrationRecord.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
can we have an associated enum for this? Earlier we sketched out

NONE = 0
PREMIGRATION = 1
MIGRATION = 2
POSTMIGRATION = 3

(0 being the default if no record is present)